cmd/kind/version/version.go

```diff
@@ -18,21 +18,24 @@ limitations under the License.
 package version
 
 import (
+	"fmt"
+	"io"
+
 	"github.com/spf13/cobra"
 )
 
 // Version is the kind CLI version
 const Version = "0.1.0-alpha"
 
 // NewCommand returns a new cobra.Command for version
-func NewCommand() *cobra.Command {
+func NewCommand(w io.Writer) *cobra.Command {
```

Comment:
cobra commands already have an io.Writer for output IIRC (?)

Reply:
> if one wants to print something custom they need to use an explicit stream.

actually...there are these wrappers.
https://godoc.org/github.com/spf13/cobra#Command.Println

feels like there are too many ways to customize this.
my vote would be to still print help and version both to stdout (one way or another).

Reply:
Yeah, I'm not sure that we should be passing stdout explicitly to this one command, there's probably a better pattern 🤔

I haven't had a chance to dig around and fight if there is a standard for which goes to what. Normally I'd use stdout and depending on the program use stderr for ... Errors 🤔

Reply:
kubeadm uses the pattern of passing the io.Writer around: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/version.go#L55

but frankly we can just `fmt.Println()` here for now.